enterprise/server/scheduling/scheduler_server/scheduler_server_test.go

```diff
@@ -143,15 +147,15 @@ func getScheduleServer(t *testing.T, userOwnedEnabled, groupOwnedEnabled bool, u
 
 func TestSchedulerServerGetPoolInfoUserOwnedDisabled(t *testing.T) {
 	s, ctx := getScheduleServer(t, false, false, "")
-	p, err := s.GetPoolInfo(ctx, "linux", "", "" /*=workflowID*/, interfaces.PoolTypeDefault)
+	p, err := s.GetPoolInfo(ctx, "linux", "", "", "" /*=workflowID*/, interfaces.PoolTypeDefault)
```

Comment:
nit: not your doing but all of these GetPoolInfo callsites should have microcomments on the empty string vals.

Reply:
Done. Also hard-coded all the `arch` values to `"amd64"` since in practice, we never pass an empty string for the arch.